modules/core/src/main/java/org/apache/ignite/spi/discovery/isolated/IsolatedDiscoverySpi.java

```diff
@@ -173,8 +173,8 @@ public class IsolatedDiscoverySpi extends IgniteSpiAdapter implements IgniteDisc
                 msg,
                 null));
 
-            // Acknowledge message must be send after initial message processed.
-            fut.listen((f) -> {
+            // Acknowledge message must be sent after initial message processed.
+            fut.listen(ignored -> {
```

Comment:
ignored -> () ?

Reply:
It's an IgniteFuture (which is a Public API) that was not changed at this PR.

Reply:
Rolled back :(